src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -1049,4 +1084,30 @@ export default class MutationTable<
             />
         );
     }
+
+    protected getMutations() {
+        let data: Mutation[][] | undefined = [];
+        if (this.props.dataStore) {
+            data = this.props.dataStore.allData;
+        } else if (this.props.data) {
+            data = this.props.data;
+        }
+        return data;
+    }
+
+    protected hasRequiredASCNProperty(property: string): boolean {
```

Comment:
We also have a method calls `hasASCNProperty ` in the mutationUtils. Are they the same?